code/modules/research/techweb/_techweb.dm

```diff
@@ -217,7 +217,7 @@
 	if(!istype(src, /datum/techweb/admin))

```

Comment:
```suggestion
	if(!istype(src, /datum/techweb/admin) && !node.starting_node)
```

Or just do this and exclude starting nodes

Reply:
It wouldn't fix the problem of future nodes giving mad dosh but it might be a good idea to have as well